client/blocks/video-editor/style.scss

```diff
@@ -0,0 +1,90 @@
+.video-editor {
+	position: relative;
+	width: 100%;
+	height: 100%;
+}
+
+.editor-media-modal .video-editor .notice {
```

Comment:
Let's pass a classname to the notice, maybe something like `.video-editor__error-notice`. This will let us simplify the selector down to:
```
.video-editor__error-notice {
```